src/frontend/js/widgets/Dashboard/components/DashboardItem/Enrollment/DashboardItemEnrollment.tsx

```diff
@@ -30,6 +30,7 @@ export const DashboardItemEnrollment = ({ enrollment }: DashboardItemCourseRunPr
             key={[enrollment.id, product.id].join('_')}
             product={product}
             courseRun={enrollment.course_run}
```

Comment:
`<ProductCertificateFooter/>` use  `courseRun` to check it's `state` and `course.code``

Reply:
Yup but enrollment is a required prop and you get courseRun from the enrollment object so we could assume to use `enrollment.course_run` instead of the `courseRun` props